File: 02_EstructurasControl/Secuenciales/Semana3_Secuenciales/Actividad Semanal/C#/Ejercicio_ActividadSemanal_6/Ejercicio_ActividadSemanal_6.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejercicio_ActividadSemanal_6
{
    internal class Program
    {
        static void Main(string[] args)
        {
            //Hallar las raíces cuadráticas.
            Console.Write("Ingrese el valor de a: ");
            double a = double.Parse(Console.ReadLine());

            Console.Write("Ingrese el valor de b: ");
            double b = double.Parse(Console.ReadLine());

            Console.Write("Ingrese el valor de c: ");
            double c = double.Parse(Console.ReadLine());

            double d = Math.Pow(b, 2) - 4 * a * c;

            double lado1 = (-b + Math.Sqrt(d)) / (2 * a);
            double lado2 = (-b - Math.Sqrt(d)) / (2 * a);

            Console.WriteLine("La primera raíz es: " + lado1);
            Console.WriteLine("La segunda raíz es: " + lado2);
            Console.ReadKey();
        }
    }
}

File: 02_EstructurasControl/Secuenciales/Semana3_Secuenciales/Actividad Semanal/Python/Ejercicio_ActividadSemanal_6.py
# Hallar las raíces cuadráticas.

a = float(input("Ingrese a: "))
b = float(input("Ingrese b: "))
c = float(input("Ingrese c: "))

discriminante = (b**2 - 4*a*c) ** 0.5

x1 = (-b + discriminante) / (2*a)
x2 = (-b - discriminante) / (2*a)

print("Raíz 1:", x1)
print("Raíz 2:", x2)